cmake/configure/configure_vectorization.cmake

```diff
@@ -0,0 +1,40 @@
+## ---------------------------------------------------------------------
+##
+## Copyright (C) 2018 by the deal.II authors
+##
+## This file is part of the deal.II library.
+##
+## The deal.II library is free software; you can use it, redistribute
+## it, and/or modify it under the terms of the GNU Lesser General
+## Public License as published by the Free Software Foundation; either
+## version 2.1 of the License, or (at your option) any later version.
+## The full text of the license can be found in the file LICENSE.md at
+## the top level directory of deal.II.
+##
+## ---------------------------------------------------------------------
+ 
+#
+# Configuration for real scalar vectorization
+#
+
+
+SET(DEAL_II_EXPAND_REAL_SCALARS_V "VectorizedArray<double,1>" "VectorizedArray<float,1>")
+SET(DEAL_II_EXPAND_FLOAT_V "VectorizedArray<float,1>")
+
+IF((${DEAL_II_COMPILER_VECTORIZATION_LEVEL} GREATER_EQUAL 1) AND DEAL_II_HAVE_SSE2)
```

Comment:
I think you can omit the `DEAL_II_HAVE_SSE2` part - this way, you also include the altivec case here; furthermore, the check `DEAL_II_COMPILER_VECTORIZATION_LEVEL` is immediately tied to `DEAL_II_HAVE_SSE2` anyway.